cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -613,6 +613,113 @@ pOperatorCertIssueCounterFile =
       ]
     ]
 
+---
+
+pAddCommitteeColdVerificationKeyOrHashOrFile :: Parser (VerificationKeyOrHashOrFile CommitteeColdKey)
```

Comment:
There is already a parser named `pCommitteeColdVerificationKeyOrHashOrFile` that returns this type. Maybe reuse it and introduce a prefix to distinguish the `add` and `remove` cases? In this way I think you don't need to add new functions to this module.

Reply:
Being explicit has two advantages:

* When you grep for the flag, it's easier to find 
* You have the ability to give different help text for the two cases.  They currently have the same help text, but we should actually make them different because they do different things.

Reply:
I agree. Explicit is better otherwise its a PITA to grep for the flag.